Source/BugsnagPlugin.h

```diff
@@ -1,16 +1,17 @@
 #ifndef BugsnagPlugin_h
 #define BugsnagPlugin_h
 
+@class Bugsnag;
+
 /**
  * Internal interface for adding custom behavior
  */
 @protocol BugsnagPlugin <NSObject>
 
 @required
 
-- (BOOL)isStarted;
-
-- (void)start;
+- (void)load;
```

Comment:
This should take a client instance as a parameter

Reply:
`BugsnagClient` is not publicly accessible and there's no point passing in the `Bugsnag` facade because it can't be instantiated. I agree that the client would ideally be passed in as a parameter to this method if we were following the spec, but the amount of work this would take means it's something we would need to address separately. This is definitely something we want to discuss once the current crop of tickets have been implemented.

Reply:
Will you be able to write your RN plugin from this? Will it just use the `Bugsnag` static to access the client?

Reply:
Yep, the React Native plugin will have to use the `Bugsnag` facade to start off with. If the deviations with `BugsnagClient` are addressed then it should be possible to alter the plugin implementation later on.